portfolio/src/main/webapp/style.css

```diff
@@ -109,6 +110,46 @@ main > .comments-section > .comments-container > :not(:last-child) {
 
 main > .comments-section .comments-list {
   flex: 2 0 0;
+  display: flex;
+  overflow-x: auto;
+}
+
+main > .comments-section .comments-list > .comment {
```

Comment:
let's try to reduce these selectors and make them as simple as possible. I believe this one could be as simple as `.comments-list .comment { ... }`

Reply:
Done!